server/controllers/erasure-search.js

```diff
@@ -0,0 +1,98 @@
+const { default: log } = require('@financial-times/n-logger');
+const operation = 'erasure-search';
+
+async function searchUserTableByUuid (db, uuid, requestedBy) {
+	try {
+		const userTableResults = await db.run('SELECT * FROM syndication.users WHERE user_id = $1;', [uuid]);
```

Comment:
could this batch instead, so the query becomes
```sql
SELECT * FROM syndication.users WHERE user_id in ['…', '…'];
```

Reply:
Apparently `IN` doesn't work with parameterised queries, but `ANY` does.

Only thing about doing that is that I would have to change how I output the results, and to be honest I don't think it's worth it for the small number of requests we'd be handling.